### PR TITLE
[machine, node] Change IO_WAIT to IO_SEND_AND_WAIT

### PR DESCRIPTION
**Pull request by snario**
Slightly more easy to reason about.

### CODE REVIEW

File: packages/machine/src/protocol/install-virtual-app.ts [hunk 1]
```diff
@@ -112,10 +110,7 @@ export const INSTALL_VIRTUAL_APP_PROTOCOL: ProtocolExecutionFlow = {
       context.outbox[0].signature2 = context.signature2; // s7
     },
 
-    Opcode.IO_SEND,
-
-    // wait for M4
```
Comment (IIIIllllIIIIllllIIIIllllIIIIllllIIIIll) [inline packages/machine/src/protocol/install-virtual-app.ts]: We should keep this comment

File: packages/node/src/node.ts [hunk 2]
```diff
@@ -226,8 +229,8 @@ export class Node {
    *     solely to the deffered promise's resolve callback.
    */
   private async handleReceivedMessage(msg: NodeMessage) {
-    if (!NODE_EVENTS[msg.event]) {
-      console.error(`Received message with unknown event type`);
+    if (!Object.values(NODE_EVENTS).includes(msg.event)) {
```
Comment (joelalejandro) [inline packages/node/src/node.ts]: I hate that TypeScript forces the use of Object.values for such a trivial task :(OI hat

Comment (snario) [reply packages/node/src/node.ts]: Yeah I noticed with the other change it was always evaluating to `true` so have to revert

Comment (joelalejandro) [reply packages/node/src/node.ts]: Yeah, I re-read it and sorry 'bout that, I was talking non-sense, lol.

**Pull request merged**